Gemfile

```diff
@@ -20,40 +20,34 @@ gem "sass-rails", "~> 6.0"
 gem "turbolinks", "~> 5"
 gem "tzinfo-data", platforms: %i[mingw mswin x64_mingw jruby]
 gem "uglifier", ">= 1.3.0"
+gem "dotenv-rails"
```

Comment:
Should `dotenv-rails` be left in the development / test group? Production usually has environment variables set directly, rather than using `.env` files.